### ISSUE TITLE
Update documents

### ISSUE THREAD

**Issue by 173210** [roxas75/rxTools#273]
This is necessary for #272.

**Comment by kot2002** [comment]
We should make TO-DO list so that we know how close the stable release is.

(e.g.) To do: 

1. README & Documents update
2. Data installation on N3DS (if needed)
3. AGB_FIRM & TWL_FIRM patches for N3DS
4. Region-free patch for N3DS
5. etc..

**Comment by 173210** [comment]
@kot2002 I suggest to make issues for them and associate a milestone for the release.
By the way, I'm considering to release a new version without common key Y as soon as possible. Please discuss at #272.

**Comment by urherenow** [comment]
Thought that was already done (removing key y). And Data installation for 3DS? N3DS is completely broken right now. Most people could care less about 2 seconds faster boot; they'd like it to simply work.

**Comment by AlbertoSONIC** [comment]
I suggest to stop doing anything and fix n3ds support... I can't even boot
rxTools on my 9.0 n3ds xl...

Il giorno Mar 27 Ott 2015 09:07 urherenow <notifications@github.com> ha
scritto:

**Comment by 173210** [comment]
@urherenow Actually the source code has been fixed, but the binary is not released yet. Data installation for 3DS is almost OK. It can generate decrypted NATIVE_FIRM with CETK and an encrypted one. The release should be done as soon as possible, so we must focus on necessary fixes.
Please discuss at #272 about the new release.
@AlbertoSONIC It works on my new 3DS. If the latest one won't work, why don't you reopen #267?

**Comment by AlbertoSONIC** [comment]
@173210 Didn't try the last one ;)

**Comment by AlbertoSONIC** [comment]
I just tried the last one, All I get from ninjhax2 is a freezed 3ds with rxtools top splash drawn....

**Comment by nastys** [comment]
I think the documents should be written for the Release Candidate. rxTools is evolving too quickly right now, so it doesn't make sense to do it now.

**Comment by b-** [comment]
I have to agree with @nastys on this. I updated the readme a number of commits back, and even since then a lot has changed with regards to cdn_firm.py alone. In the past two or three months alone the method with which you go from source code to a bootable installation have changed massively. This issue should definitely remain open and with high importance, but it's not really possible in my eyes to fully close it until we have something more stable and standardized.

**Comment by 173210** [comment]
@AlbertoSONIC It's time to open #267 again, isn't it?

**Comment by nastys** [comment]
A new manual is W.I.P., so I guess this can be closed now?

**Comment by 173210** [comment]
Probably we should wait the manual gets ready for release.

**Comment by nastys** [comment]
OK

**Comment by urherenow** [comment]
The manual is very good as it is right now. Even if it's not ready for release, It wouldn't hurt to add the current copy to the git. Maybe in a Docs folder.

**Comment by nastys** [comment]
No yet, but don't worry: the final version will be finished soon, and it will be much better than you think :wink:
